api/base.py

```diff
@@ -96,9 +94,12 @@ def __init__(self, request=None, response=None):
         elif drone_request:
             self.superuser_request = True
         else:
-            user = self.app.db.users.find_one({'_id': self.uid}, ['root', 'wheel'])
+            user = self.app.db.users.find_one_and_update({'_id': self.uid}, {'$set': {'lastseen': request_start}}, ['root', 'wheel'])
```

Comment:
That's what that does right now. Open to discussion. I was thinking we'd try it and see how it performs, but on second thought, I will add a separate query to the remote token block.

Renaming ```firstsseen``` and ```lastseen``` to more appropriate ```firstlogin``` and ```lastlogin```, respectively.